clrsvsim/simulator.py

```diff
@@ -138,7 +139,11 @@ def make_split_read(read, breakpoint, clip_left, hard_clip_threshold=1.0, sequen
             split_seq[:breakpoint] = sequence[-breakpoint:]
         else:
             split_seq[breakpoint:] = sequence[:read.rlen - breakpoint]
+        qual = split_read.qual
         split_read.seq = ''.join(split_seq)
+        if len(qual) < len(split_seq):
```

Comment:
this check is not needed, if `len(qual) >= len(split_seq)` then `DEFAULT_QUAL * (len(split_seq) - len(qual))` will just evaluate to an empty string.